### ISSUE TITLE
Unable to use spyder 3 icon set with Spyder 5 (installed with the Windows installer)

### ISSUE THREAD

**Issue by danhamill** [spyder-ide/spyder#15347]
### Issue Report Checklist

* [ X] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [ X] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [ ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [X ] Tried basic troubleshooting (if a bug/error)
    * [ X] Restarted Spyder
    * [ X] Uninstall and reinstall
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [ X] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
I installed Spyder 5.0.1 using the standalone installer and the program worked.  I changed the icon appearance to spyder3 and restarted the program.  The spyder loading window with shows on my desktop and quits after loading toolbars message

How can you reset spyder defaults in the standalone installer?
### What steps reproduce the problem?

1. Install Spyder 5.0.1 using standalone installer
2. Change icons to spyder 3
3.  Restart spyder

### What is the expected output? What do you see instead?

Sypder loads correctly 

### Paste Traceback/Error Below (if applicable)

```python-traceback

PASTE TRACEBACK HERE

```

## Versions

* Spyder version: 5.0.1 (standalone installer)
* Python version: 
* Qt version: 
* PyQt version: 
* Operating System name/version: 

### Dependencies

```
PASTE DEPENDENCIES HERE
```

**Comment by ccordoba12** [comment]
Hey @danhamill, thanks for reporting. For now the only way to reset Spyder to its defaults is by removing this directory

    C:\Users\<your user>\.spyder-py3

But we'll try to improve that in a future release.

**Comment by ccordoba12** [comment]
@dalthviz, could you reproduce this problem?

**Comment by danhamill** [comment]
hmm.  I tried deleting the folder in my user directory, but the program still wont launch.  When I attempt to start it, the loading window closes after it displays "loading toolbars".

**Comment by danhamill** [comment]
```
C:\Program Files\Spyder\Python>python ..\Spyder.launch.pyw
No QCoreApplication instance found. Application patches not applied. You have to call load_stylesheet function after instantiation of QApplication to take effect.
Traceback (most recent call last):
  File "..\pkgs\spyder\app\mainwindow.py", line 2161, in create_window
    main.setup()
  File "..\pkgs\spyder\app\mainwindow.py", line 903, in setup
    plugin_instance = plugin_class(self, configuration=CONF)
  File "..\pkgs\spyder\plugins\preferences\plugin.py", line 58, in __init__
    super().__init__(parent, configuration)
  File "..\pkgs\spyder\api\plugins.py", line 833, in __init__
    container.setup()
  File "..\pkgs\spyder\plugins\preferences\widgets\container.py", line 107, in setup
    icon=self.create_icon('configure'),
  File "..\pkgs\spyder\api\widgets\mixins.py", line 573, in create_icon
    return ima.icon(name)
  File "..\pkgs\spyder\utils\icon_manager.py", line 375, in icon
    directory=self._resource['directory'])
  File "..\pkgs\qtawesome\__init__.py", line 201, in load_font
    return _instance().load_font(prefix, ttf_filename, charmap_filename, directory)
  File "..\pkgs\qtawesome\__init__.py", line 76, in _instance
    ('mdi', 'materialdesignicons-webfont.ttf',
  File "..\pkgs\qtawesome\iconic_font.py", line 214, in __init__
    self.load_font(*fargs)
  File "..\pkgs\qtawesome\iconic_font.py", line 270, in load_font
    os.path.join(directory, ttf_filename)))
qtawesome.iconic_font.FontError: Font at 'C:\Program Files\Spyder\pkgs\qtawesome\fonts\fontawesome4.7-webfont.ttf' appears to be empty. If you are on Windows 10, please read https://support.microsoft.com/en-us/kb/3053676 to know how to prevent Windows from blocking the fonts that come with QtAwesome.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "..\Spyder.launch.pyw", line 35, in <module>
    main()
  File "..\pkgs\spyder\app\start.py", line 213, in main
    mainwindow.main(options, args)
  File "..\pkgs\spyder\app\mainwindow.py", line 2311, in main
    mainwindow = create_window(app, splash, options, args)
  File "..\pkgs\spyder\app\mainwindow.py", line 2163, in create_window
    if main.console is not None:
  File "..\pkgs\spyder\app\mainwindow.py", line 1180, in __getattr__
    return self.get_plugin(self._INTERNAL_PLUGINS_MAPPING[attr])
  File "..\pkgs\spyder\app\mainwindow.py", line 170, in get_plugin
    'Plugin "{}" not found!'.format(plugin_name))
spyder.api.exceptions.SpyderAPIError: Plugin "internal_console" not found!
```

**Comment by dalthviz** [comment]
Hi @danhamill so your issue is that using the spyder 3 icon set Spyder doesn't launch, right? As a workaround for now, I think you can change your config manually by changing the `spyder.ini` file inside the preferences directory (which is the one described above in a path like `C:\Users\<your user>\.spyder-py3`). There you can find a property call `icon_theme` and change the value to be `spyder 2`.

Also, thanks for the traceback! I think the font is not being able to load in your system, is there any security policy configured preventing third-party fonts in your setup?

**Comment by dalthviz** [comment]
@ccordoba12 @danhamill I was unable to reproduce this :/

Also @ccordoba12 when I change from `Spyder 3` to `Spyder 2` I still see some icons from the `Spyder 3` theme, is that correct?:

![image](https://user-images.githubusercontent.com/16781833/115275953-bca86c80-a142-11eb-9714-fd67ed228fbc.png)

**Comment by steff456** [comment]
> Also when I change from Spyder 3 to Spyder 2 I still see some icons from the Spyder 3 theme, is that correct?:

Yes, custom icons are now loaded from Spyder 3 set of icons because in a lot of cases there wasn't any icon for those.

**Comment by ccordoba12** [comment]
This is solved now in Spyder 5.4.0 with a recent enough Windows 10 version.
